app/services/booking_request_creator.rb

```diff
@@ -38,7 +38,9 @@ def visitors_step_params(step)
       visitor_last_name: step.last_name,
       visitor_date_of_birth: step.date_of_birth,
       visitor_email_address: step.email_address,
-      visitor_phone_no: step.phone_no
+      visitor_phone_no: step.phone_no,
+      override_email_checks: step.override_email_checks,
+      email_override: step.email_override
```

Comment:
Yeah, on reflection they are named confusingly.  `email_override` is the category of the error and override_email_checks is the flag.  I'll rename `email_override`.